AbletonMCP_Remote_Script/handlers/devices.py

```
@@ -0,0 +1,1342 @@
+
+    for v in range(lo, hi + 1):
+        try:
+            disp = param.str_for_value(float(v))
+            if disp is None:
+                continue
+            disp_norm = _normalize_display(disp)
+            if disp_norm == target_norm:
+                if ctrl:
+                    ctrl.log_message("  MATCH at v={0}".format(v))
+                return float(v)
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)